src/plugins/plugin.tooltip.js

```diff
@@ -671,7 +671,7 @@ export class Tooltip extends Element {
 		}
 
 		if (changed && options.custom) {
-			options.custom.call(me);
+			options.custom.call(me, me);
```

Comment:
I wonder if we should make the parameter follow the same pattern as [scriptable options](https://www.chartjs.org/docs/latest/general/options.html#scriptable-options). I.e. pass a context object `{chart, tooltipModel}`. What do you think @etimberg @kurkle?

Reply:
We could make that change for v3